pkg/services/ngalert/api/tooling/definitions/cortex-ruler.go

```diff
@@ -152,6 +152,7 @@ import (
 
 // swagger:parameters RoutePostNameRulesConfig RoutePostNameGrafanaRulesConfig RoutePostRulesGroupForExport
 type NamespaceConfig struct {
+	// The UID of the rule folder
```

Comment:
IIRC this is shared between Lotex and Grafana APIs, and in the former case it's name, not UID.